Custom-Capturer-Camera2/README.md

```diff
@@ -0,0 +1,85 @@
+# Custom Video Capturer (camera2 interface)
+Sample app shows how to use the custom capturer using camera2 package. 
+Note: You need to have a minimum api level 21 to run this sample app
+
+## Using a custom video capturer
+
+The MainActivity class shows how you can use a custom video capturer for a publisher. After
+instantiating a Publisher object, the code sets a custom video capturer as shown below.
+
+```java
+     mPublisher = new Publisher.Builder(MainActivity.this)
+                    .name("publisher")
+                    .capturer(new CustomVideoCapturerCamera2(MainActivity.this, Publisher.CameraCaptureResolution.MEDIUM,                           Publisher.CameraCaptureFrameRate.FPS_30))
+                    .renderer(new InvertedColorsVideoRenderer(MainActivity.this)).build();
```

Comment:
I think we should not use a special renderer in this sample.
We are just showing how to use Camera2 capturer